<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>puzzle</class>
 <widget class="QWidget" name="puzzle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="fondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/fondo_puzzle.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IMAGEN_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IMAGEN_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IMAGEN_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IMAGEN_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IMAGEN_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IMAGEN_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IMAGEN_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="IMAGEN_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/carretera.png</pixmap>
   </property>
  </widget>
  <widget class="QLCDNumber" name="vidas">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="im_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/carta_1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="im_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/carta_3.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="im_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/carta_2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="im_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/carta_2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="im_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/carta_4.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="im_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/carta_3.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="im_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/carta_1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="im_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/carta_4.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="in_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/inter_3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="in_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/inter_3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="in_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/inter_3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="in_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/inter_3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="in_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/inter_3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="in_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/inter_3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="in_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/inter_3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="in_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/inter_3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imagen.qrc">:/intentos_at.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>fondo</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>vidas</zorder>
  <zorder>im_1</zorder>
  <zorder>im_2</zorder>
  <zorder>im_3</zorder>
  <zorder>im_4</zorder>
  <zorder>im_5</zorder>
  <zorder>im_6</zorder>
  <zorder>im_7</zorder>
  <zorder>im_8</zorder>
  <zorder>in_1</zorder>
  <zorder>in_2</zorder>
  <zorder>in_3</zorder>
  <zorder>in_4</zorder>
  <zorder>in_5</zorder>
  <zorder>in_6</zorder>
  <zorder>in_7</zorder>
  <zorder>in_8</zorder>
  <zorder>IMAGEN_1</zorder>
  <zorder>IMAGEN_2</zorder>
  <zorder>IMAGEN_3</zorder>
  <zorder>IMAGEN_4</zorder>
  <zorder>IMAGEN_5</zorder>
  <zorder>IMAGEN_6</zorder>
  <zorder>IMAGEN_7</zorder>
  <zorder>IMAGEN_8</zorder>
 </widget>
 <resources>
  <include location="imagen.qrc"/>
 </resources>
 <connections/>
</ui>
